resources/ext/cli/config-migration

```diff
@@ -99,6 +99,7 @@ function replaceline {
   tmp="$3.tmp.$(date +%s)"
   sed "s/$1/$(echo "$2" | sed -e 's/[\/&]/\\&/g')/g" "$3" > "$tmp"
   mv "$tmp" "$3"
+  chmod 644 "$3"
```

Comment:
All good points, thanks.  I'd only been trying to preserve backward compatibility with how this was working before just after package install, allowing the service to still be able to boot properly.  At initial package install, the config files have 644 permissions and are owned by root:root.

Assuming these config files may have sensitive data, it might be worth a broader investigation into ensuring that all config files are installed with service owner permissions and a 0640 mode - along with your additional recommendations about how to ensure that world-accessibility isn't, even briefly, re-granted as these data migrations are occurring.  I'm not planning on following up on this piece further at this point but might be worth pursuing with the Release Engineering team further.  I take it that they may own (or at least had initially written) some of these scripts?